sea-orm-macros/src/derives/from_query_result.rs

```diff
@@ -42,40 +101,54 @@ pub fn expand_derive_from_query_result(
             })
         }
     };
-    let mut field = Vec::with_capacity(fields.len());
 
-    for parsed_field in fields.into_iter() {
-        let mut skip = false;
+    let mut fields = Vec::with_capacity(parsed_fields.len());
+    for parsed_field in parsed_fields.into_iter() {
+        let mut typ = ItemType::Normal;
         for attr in parsed_field.attrs.iter() {
             if !attr.path().is_ident("sea_orm") {
                 continue;
             }
             if let Ok(list) = attr.parse_args_with(Punctuated::<Meta, Comma>::parse_terminated) {
                 for meta in list.iter() {
-                    skip = meta.exists("skip");
+                    if meta.exists("skip") {
+                        typ = ItemType::Skipped;
+                    } else if meta.exists("nested") {
+                        typ = ItemType::Nested;
+                    }
                 }
             }
         }
         let ident = format_ident!("{}", parsed_field.ident.unwrap().to_string());
-        field.push(FromQueryResultItem { skip, ident });
+        fields.push(FromQueryResultItem { typ, ident });
     }
     let (impl_generics, ty_generics, where_clause) = generics.split_for_impl();
 
+    let ident_try_init: Vec<_> = fields.iter().map(TryFromQueryResultCheck).collect();
+    let ident_try_assign: Vec<_> = fields.iter().map(TryFromQueryResultAssignment).collect();
+
     Ok(quote!(
         #[automatically_derived]
         impl #impl_generics sea_orm::FromQueryResult for #ident #ty_generics #where_clause {
-            fn from_query_result(row: &sea_orm::QueryResult, pre: &str) -> std::result::Result<Self, sea_orm::DbErr> {
+            fn from_query_result(row: &sea_orm::QueryResult, pre: &str) -> Result<Self, sea_orm::DbErr> {
```

Comment:
please use std::result::Result

Reply:
Valid criticism. The correct way is to use `::std::..` or `::sea_orm::..` everywhere, so I will adjust it to that.